toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -255,9 +258,13 @@ func (r *RpmRepoCloner) initializeRepoDefinitions(repoDefinitions []string) (err
 	// Add each previously existing repofile to the end of the new file, then delete the original.
 	// We want to try our custom mounted repos before reaching out to the upstream servers.
 	for _, originalRepoFilePath := range existingRepoFiles {
-		err = appendRepoFile(originalRepoFilePath, dstFile)
-		if err != nil {
-			return
+		if !r.disableMarinerRepos {
+			err = appendRepoFile(originalRepoFilePath, dstFile)
+			if err != nil {
+				return
+			}
+		} else {
+			logger.Log.Infof("Disabling repositories listed in %s", originalRepoFilePath)
```

Comment:
>logger.Log.Infof("Disabling repositories listed in %s", originalRepoFilePath) [](http://example.com/codeflow?start=3&length=77)

The users are not familiar with the files we're removing here, so these messages may be confusing to them and just add unnecessary noise. I'd punt this down to a debug log.

For the user we can just print one generic "Disabling default upstream RPM repositories" message or something along these lines.

Reply:
updated